README.md

```diff
@@ -1,94 +1,250 @@
+1. Create and move into a working directory for your node client data
 
-```bash
-# Installs the LUKSO CLI and prompts user to select its Consensus and Execution clients.
-# Install also detects if you have any pre-installed client and confirms an override to a newer version in case needed.
+```sh
+# Exchange [folder_name] with the folder name you want
+$ mkdir [folder_name] && cd ./[folder_name]
+```
+
+2. Initialize the working directory
+
+```sh
+# Downloads all network configs from
+$ lukso init
+```
+
+3. Install choosen LUKSO node clients into the working directory
+
+```sh
+# Installing Execution Chain, Beacon Chain, and Validator Client
+# Might need admin access by typing `sudo` in front of the command
 $ lukso install
+```
 
-# Installs clients and agrees with Terms & Conditions automatically
-$ lukso install --agree-terms
+## LUKSO CLI Usage
+
+```sh
+lukso [global options] [command] [command options] [arguments...] [global options]
 ```
 
-## How to view logs
+> Global Options can be placed at the beginning or end of the terminal entry
+
+## Available Commands
+
+| Command   | Description                                                                                      |
+| --------- | ------------------------------------------------------------------------------------------------ |
+| install   | Installs choosen LUKSO node clients (Execution, Beacon, Validator) and their binary dependencies |
+| init      | Initializes the node working directory, it's structure, and network configuration                |
+| update    | Updates all or specific LUKSO node clients in the working directory to the newest version        |
+| start     | Starts all or specific LUKSO node clients and connects to the specified network                  |
+| stop      | Stops all or specific LUKSO node clients that are currently running                              |
+| log       | Listens to all log events from a specific client in the current terminal window                  |
+| status    | Shows the LUKSO node client processes that are currently running                                 |
+| reset     | Resets all or specific client data directories and logs excluding the validator keys             |
+| validator | Manages the LUKSO validator keys including their initialization and deposits                     |
+| version   | Display the version of the LUKSO CLI Tool that is currently installed                            |
+| help, h   | Shows the full list of commands, global options, and their usage                                 |
+
+## Available Global Options
+
+Global options can be added behind a command to allow different modifications to its execution.
+
+| Global Option                    | Description                                        |
+| -------------------------------- | -------------------------------------------------- |
+| --accept-terms-of-use            | Automatically accept upcoming terms of use         |
```

Comment:
shouldn't be a global option. and it should be `--accept-terms`
lets first get the cli right. as we need to update this after

Reply:
will add it into a global option ticket